2021/03/18 20:01:46 [error] 31#31: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.3:15672/", host: "localhost"
2021/03/18 20:01:49 [error] 31#31: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.4:15672/", host: "localhost"
2021/03/18 20:01:52 [error] 31#31: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.2:15672/", host: "localhost"
2021/03/18 20:01:52 [error] 31#31: *1 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "localhost", referrer: "http://localhost/"
2021/03/18 20:01:55 [error] 31#31: *1 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://stream_rabbit_http/", host: "localhost"
2021/03/18 20:01:55 [error] 31#31: *1 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "localhost", referrer: "http://localhost/"
2021/03/18 20:02:33 [error] 31#31: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.2:15672/", host: "localhost"
2021/03/18 20:02:36 [error] 31#31: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.4:15672/", host: "localhost"
2021/03/18 20:02:39 [error] 31#31: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.3:15672/", host: "localhost"
2021/03/18 20:02:39 [error] 31#31: *1 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "localhost", referrer: "http://localhost/"
2021/03/18 20:02:49 [error] 31#31: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.2:15672/", host: "localhost"
2021/03/18 20:02:52 [error] 31#31: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.4:15672/", host: "localhost"
2021/03/18 20:02:55 [error] 31#31: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.29.0.3:15672/", host: "localhost"
2021/03/18 20:02:55 [error] 31#31: *1 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "localhost", referrer: "http://localhost/"
2021/03/18 20:10:25 [error] 24#24: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.3:15672/", host: "localhost"
2021/03/18 20:10:25 [error] 24#24: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:15672/", host: "localhost"
2021/03/18 20:10:25 [error] 24#24: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.5:15672/", host: "localhost"
2021/03/18 20:10:26 [error] 24#24: *1 no live upstreams while connecting to upstream, client: 172.30.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "localhost", referrer: "http://localhost/"
2021/03/18 20:10:27 [error] 24#24: *1 no live upstreams while connecting to upstream, client: 172.30.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://stream_rabbit_http/", host: "localhost"
2021/03/18 20:10:28 [error] 24#24: *1 no live upstreams while connecting to upstream, client: 172.30.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "localhost", referrer: "http://localhost/"
2021/03/18 20:11:54 [error] 24#24: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.5:15672/", host: "ubuntu"
2021/03/18 20:11:54 [error] 24#24: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:15672/", host: "ubuntu"
2021/03/18 20:11:54 [error] 24#24: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.3:15672/", host: "ubuntu"
2021/03/18 20:11:55 [error] 24#24: *5 no live upstreams while connecting to upstream, client: 172.30.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "ubuntu", referrer: "http://ubuntu/"
2021/03/18 20:12:58 [error] 24#24: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.206.194, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.5:15672/", host: "192.168.206.194"
2021/03/18 20:12:58 [error] 24#24: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.206.194, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.3:15672/", host: "192.168.206.194"
2021/03/18 20:12:58 [error] 24#24: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.206.194, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:15672/", host: "192.168.206.194"
2021/03/18 20:12:58 [error] 24#24: *9 no live upstreams while connecting to upstream, client: 192.168.206.194, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "192.168.206.194", referrer: "http://192.168.206.194/"
2021/03/18 20:14:07 [error] 24#24: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.206.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.5:15672/", host: "ubuntu"
2021/03/18 20:14:07 [error] 24#24: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.206.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:15672/", host: "ubuntu"
2021/03/18 20:14:07 [error] 24#24: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.206.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.3:15672/", host: "ubuntu"
2021/03/18 20:14:44 [error] 24#24: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.206.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.5:15672/", host: "ubuntu"
2021/03/18 20:14:44 [error] 24#24: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.206.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.3:15672/", host: "ubuntu"
2021/03/18 20:14:44 [error] 24#24: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.206.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:15672/", host: "ubuntu"
2021/03/18 20:14:44 [error] 24#24: *13 no live upstreams while connecting to upstream, client: 192.168.206.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "ubuntu", referrer: "http://ubuntu/"
2021/03/18 20:15:15 [error] 24#24: *20 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.206.194, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.5:15672/", host: "192.168.206.194"
2021/03/18 20:15:15 [error] 24#24: *20 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.206.194, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:15672/", host: "192.168.206.194"
2021/03/18 20:15:15 [error] 24#24: *20 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.206.194, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.3:15672/", host: "192.168.206.194"
2021/03/18 20:15:16 [error] 24#24: *20 no live upstreams while connecting to upstream, client: 192.168.206.194, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "192.168.206.194", referrer: "http://192.168.206.194/"
2021/03/18 20:15:25 [error] 24#24: *20 no live upstreams while connecting to upstream, client: 192.168.206.194, server: , request: "GET / HTTP/1.1", upstream: "http://stream_rabbit_http/", host: "192.168.206.194"
2021/03/18 20:15:25 [error] 24#24: *20 no live upstreams while connecting to upstream, client: 192.168.206.194, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "192.168.206.194", referrer: "http://192.168.206.194/"
2021/03/18 20:16:07 [error] 24#24: *24 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.5:15672/", host: "localhost"
2021/03/18 20:16:07 [error] 24#24: *24 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.3:15672/", host: "localhost"
2021/03/18 20:16:07 [error] 24#24: *24 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.30.0.2:15672/", host: "localhost"
2021/03/18 20:16:08 [error] 24#24: *24 no live upstreams while connecting to upstream, client: 172.30.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "localhost", referrer: "http://localhost/"
2021/03/19 14:09:20 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.19.0.5:15672/", host: "localhost"
2021/03/19 14:09:20 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.19.0.3:15672/", host: "localhost"
2021/03/19 14:09:20 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.19.0.2:15672/", host: "localhost"
2021/03/19 14:09:21 [error] 30#30: *1 no live upstreams while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "localhost"
2021/03/19 14:22:36 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: 0.0.0.0:5000, upstream: "172.21.0.3:5672", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 14:22:36 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: 0.0.0.0:5000, upstream: "172.21.0.2:5672", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 14:22:36 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: 0.0.0.0:5000, upstream: "172.21.0.5:5672", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 14:22:36 [error] 30#30: *5 no live upstreams while connecting to upstream, client: 172.21.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 14:22:36 [error] 30#30: *6 no live upstreams while connecting to upstream, client: 172.21.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 14:22:36 [error] 30#30: *7 no live upstreams while connecting to upstream, client: 172.21.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 14:22:36 [error] 30#30: *8 no live upstreams while connecting to upstream, client: 172.21.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 14:22:36 [error] 30#30: *9 no live upstreams while connecting to upstream, client: 172.21.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 14:22:36 [error] 30#30: *10 no live upstreams while connecting to upstream, client: 172.21.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 14:22:36 [error] 30#30: *11 no live upstreams while connecting to upstream, client: 172.21.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 14:22:36 [error] 30#30: *12 no live upstreams while connecting to upstream, client: 172.21.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 14:22:36 [error] 30#30: *13 no live upstreams while connecting to upstream, client: 172.21.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 14:22:36 [error] 30#30: *14 no live upstreams while connecting to upstream, client: 172.21.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 14:22:36 [error] 30#30: *15 no live upstreams while connecting to upstream, client: 172.21.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 14:22:59 [error] 30#30: *16 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.21.0.3:15672/", host: "localhost:15000"
2021/03/19 14:22:59 [error] 30#30: *16 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.21.0.2:15672/", host: "localhost:15000"
2021/03/19 14:22:59 [error] 30#30: *16 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.21.0.5:15672/", host: "localhost:15000"
2021/03/19 14:23:00 [error] 30#30: *16 no live upstreams while connecting to upstream, client: 172.21.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "localhost:15000"
2021/03/19 14:24:16 [error] 30#30: *20 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.21.0.5:15672/", host: "localhost:15000"
2021/03/19 14:24:16 [error] 30#30: *20 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.21.0.2:15672/", host: "localhost:15000"
2021/03/19 14:24:16 [error] 30#30: *20 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.21.0.3:15672/", host: "localhost:15000"
2021/03/19 17:00:16 [emerg] 1#1: host not found in upstream "rabbitmq2:5672" in /etc/nginx/nginx.conf:8
2021/03/19 17:08:38 [emerg] 1#1: host not found in upstream "rabbitmq2:5672" in /etc/nginx/nginx.conf:8
2021/03/19 17:17:32 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.23.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.23.0.2:15672/", host: "localhost:15000"
2021/03/19 17:17:32 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.23.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.23.0.3:15672/", host: "localhost:15000"
2021/03/19 17:17:32 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.23.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.23.0.4:15672/", host: "localhost:15000"
2021/03/19 17:17:51 [error] 30#30: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.23.0.1, server: 0.0.0.0:5000, upstream: "172.23.0.2:5672", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 17:17:51 [error] 30#30: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.23.0.1, server: 0.0.0.0:5000, upstream: "172.23.0.3:5672", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 17:17:51 [error] 30#30: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.23.0.1, server: 0.0.0.0:5000, upstream: "172.23.0.4:5672", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 17:17:51 [error] 30#30: *9 no live upstreams while connecting to upstream, client: 172.23.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 17:17:51 [error] 30#30: *10 no live upstreams while connecting to upstream, client: 172.23.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 17:17:51 [error] 30#30: *11 no live upstreams while connecting to upstream, client: 172.23.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 17:17:51 [error] 30#30: *12 no live upstreams while connecting to upstream, client: 172.23.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 17:17:51 [error] 30#30: *13 no live upstreams while connecting to upstream, client: 172.23.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 17:17:51 [error] 30#30: *14 no live upstreams while connecting to upstream, client: 172.23.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 17:17:51 [error] 30#30: *15 no live upstreams while connecting to upstream, client: 172.23.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 17:17:51 [error] 30#30: *16 no live upstreams while connecting to upstream, client: 172.23.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 17:17:51 [error] 30#30: *17 no live upstreams while connecting to upstream, client: 172.23.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 17:17:51 [error] 30#30: *18 no live upstreams while connecting to upstream, client: 172.23.0.1, server: 0.0.0.0:5000, upstream: "stream_amqp", bytes from/to client:0/0, bytes from/to upstream:0/0
2021/03/19 18:12:56 [error] 31#31: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.24.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.24.0.2:15672/", host: "localhost:5000"
2021/03/19 18:12:56 [error] 31#31: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.24.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.24.0.3:15672/", host: "localhost:5000"
2021/03/19 18:12:56 [error] 31#31: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.24.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.24.0.4:15672/", host: "localhost:5000"
2021/03/19 18:12:57 [error] 31#31: *1 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://stream_rabbit_http/favicon.ico", host: "localhost:5000"
2021/03/19 18:25:07 [error] 31#31: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.24.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.24.0.4:15672/", host: "localhost:5000"
2021/03/19 18:25:07 [error] 31#31: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.24.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.24.0.3:15672/", host: "localhost:5000"
2021/03/19 18:25:07 [error] 31#31: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.24.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.24.0.2:15672/", host: "localhost:5000"
2021/03/19 18:32:03 [error] 32#32: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.25.0.2:15672/", host: "localhost:5000"
2021/03/19 18:32:03 [error] 32#32: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.25.0.3:15672/", host: "localhost:5000"
2021/03/19 18:32:03 [error] 32#32: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.25.0.4:15672/", host: "localhost:5000"
2021/03/19 18:32:48 [error] 32#32: *5 recv() failed (104: Connection reset by peer) while proxying and reading from upstream, client: 172.25.0.1, server: 0.0.0.0:15000, upstream: "172.25.0.2:5672", bytes from/to client:682/8, bytes from/to upstream:8/682
2021/03/19 18:32:56 [error] 32#32: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.25.0.4:15672/", host: "localhost:5000"
2021/03/19 18:32:56 [error] 32#32: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.25.0.3:15672/", host: "localhost:5000"
2021/03/19 18:32:56 [error] 32#32: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.25.0.2:15672/", host: "localhost:5000"
2021/03/19 18:33:36 [error] 32#32: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.25.0.4:15672/", host: "localhost:5000"
2021/03/19 18:33:36 [error] 32#32: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.25.0.2:15672/", host: "localhost:5000"
2021/03/19 18:33:36 [error] 32#32: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.25.0.3:15672/", host: "localhost:5000"
2021/03/19 18:33:46 [error] 32#32: *1 no live upstreams while connecting to upstream, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://stream_rabbit_http/", host: "localhost:5000"
2021/03/19 18:37:28 [emerg] 1#1: "proxy_timeout" directive is not allowed here in /etc/nginx/nginx.conf:39
2021/03/19 19:11:29 [emerg] 1#1: "proxy_timeout" directive is not allowed here in /etc/nginx/nginx.conf:39
2021/03/19 19:22:39 [emerg] 1#1: "proxy_timeout" directive is not allowed here in /etc/nginx/nginx.conf:36
2021/03/19 19:25:49 [emerg] 1#1: "proxy_timeout" directive is not allowed here in /etc/nginx/nginx.conf:36
